packages/rocketchat-ui-flextab/client/tabs/userEdit.js

```diff
@@ -32,26 +35,50 @@ Template.userEdit.events({
 	'click .cancel'(e, t) {
 		e.stopPropagation();
 		e.preventDefault();
-		return t.cancel(t.find('form'));
+		Template.instance().roles.set([]);
+		t.cancel(t.find('form'));
+	},
+
+	'click .remove-role'(e) {
+		e.stopPropagation();
+		e.preventDefault();
+		let roles = Template.instance().roles.get();
+		roles = roles.filter(el => el !== this.valueOf());
+		Template.instance().roles.set(roles);
```

Comment:
you can add a second parameter (`t`) on this event and use that here instead of `Template.instance()`